learning_resources/models.py

```diff
@@ -179,7 +180,7 @@ class LearningResource(TimestampedModel):
     )
     learning_format = ArrayField(
         models.CharField(max_length=24, db_index=True),
-        default=list,
+        default=[LearningResourceFormat.online.name],
```

Comment:
This should be a callable instead because otherwise the value is mutable and shared everywhere.
```suggestion
        default=lambda: [LearningResourceFormat.online.name],
```